rpmio/macro.c

```diff
@@ -1145,6 +1145,29 @@ static size_t doVerbose(MacroBuf mb, rpmMacroEntry me, ARGV_t argv)
     return 0;
 }
 
+static size_t doShescape(MacroBuf mb, rpmMacroEntry me, ARGV_t argv)
+{
+    char *result, *dst;
+    const char *src = argv[1];
+
+    result = dst = xmalloc(strlen(src) * 4 + 3);
```

Comment:
Are we sure this won’t integer overflow?

Reply:
Not on any input that we care about.